lib/services/plugins/cordova-project-plugins-service.ts

```diff
@@ -602,7 +603,11 @@ export class CordovaProjectPluginsService extends NpmPluginsServiceBase implemen
 			pluginName = pluginName.toLowerCase();
 			let isConsoleInteractive = helpers.isInteractive();
 			let allInstalledPlugins = this.getInstalledPluginsForConfiguration();
-			let installedPluginInstances = _.filter(allInstalledPlugins, (plugin: IPlugin) => plugin.data.Name.toLowerCase() === pluginName || plugin.data.Identifier.toLowerCase() === pluginName);
+			let installedPluginInstances = _(allInstalledPlugins)
+				.filter((plugin: IPlugin) => plugin.data.Name.toLowerCase() === pluginName || plugin.data.Identifier.toLowerCase() === pluginName)
+				.uniqBy((plugin: IPlugin) => plugin.data.Version)
```

Comment:
When we add plugin which is already added in more than one configurations with same version, we will get a warning `Plugin 'nl.x-services.plugins.socialsharing' is enabled with different versions in your project configurations.` because in `allInstalledPlugins ` there will be more than one instances of the plugin with the same version and that's why we need to filter the installed plugins by version. If the plugin is installed in more than one configurations with same version in the instance that is left after the filtering the `configurations` property will contain all the configurations of the plugin.